docs/docs/ref-03-component-specs.md

```diff
@@ -10,6 +10,8 @@ next: tags-and-attributes.html
 
 When creating a component class by invoking `React.createClass()`, you should provide a specification object that contains a `render` method and can optionally contain other lifecycle methods described here.
 
+Note: If use ES6 classes you can extend your class from `React.Component` and then `props` are passed in to the constructor, `propTypes` and `defaultProps` should be static members and `getInitialState` is not called (instead you should set the initial state on `this.state` in your constructor). [More on this](http://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#es6-classes)
+
```

Comment:
A few things:

- Let's make this a blockquote with "Note:" on its own line. This is the pattern we use elsewhere.
- We should link to http://facebook.github.io/react/docs/reusable-components.html#es6-classes (but it should not include the facebook.github.io, just the /react/...)
- I actually think it might be easier to just include a link there and not go into the details here (for now).
- Let's try something like this:

> Note:
> It is also possible to use plain JavaScript classes as component classes. These classes can implement most of the same methods, though there are some differences. For more information about these differences, please read our documentation about [ES6 classes](/react/docs/reusable-components.html#es6-classes).